src/nextjs-front/components/chat/DirectMessages.tsx

```diff
@@ -68,7 +68,7 @@ const DirectMessages: React.FC<{ viewParams: Object }> = ({ viewParams }) => {
     /* Sorts from most recent */
     dms.sort(
       (a: DirectMessage, b: DirectMessage) =>
-        b.updatedAt.valueOf() - a.updatedAt.valueOf()
+        new Date(b.updatedAt).valueOf() - new Date(a.updatedAt).valueOf()
```

Comment:
Prefer `Date.getTime()` to get the epoch of a Javascript date.
This is functionally equivalent to `valueOf` but [according to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/valueOf) `getTime` should be used in code directly (more explicit)

Reply:
Ty! Much better too!